packages/push-notifications/src/PushNotificationsModuleConfig.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * PushNotificationsModuleConfigOptions defines the interface for the options of the PushNotificationsModuleConfig class.
+ */
+export interface PushNotificationsModuleConfigOptions {
+  /**
+   * The provider to use - apns or fcm
+   *
+   * @default undefined
+   */
+  provider?: 'fcm' | 'apns'
```

Comment:
I tried using the modules, and you can now do this:

`new PushNotificationsFcmModule({ provider: 'apns' })`

which is a bit weird.

Reply:
This is still possible.

Reply:
you can put whatever object as options. I put that so that its clear that one can pass options, but they aren't used in the base implementation. This way you can set whatever optionsyou want/need for your own use case (well, one can). Alternatively, options could be removed entirely from the implementation which is also fine